src/infrastructure/database/repositories/games.repository.spec.js

```diff
@@ -451,4 +451,172 @@ describe("GamesRepository", function () {
       });
     });
   });
+
+  describe(".getTrendingGames returns an array of trending games based on the passed in arguments.", function () {
+    describe("If a specific time period is passed in", function () {
+      beforeAll(async function () {
+        jasmine.clock().mockDate(new Date());
+
+        this.databaseClient = await initiateInMemoryDatabase(["games"]);
+
+        await this.databaseClient.insertMany("games", getTrendingGamesMockData());
+
+        const gamesRepo = new GamesRepository(this.databaseClient);
+
+        this.result = await gamesRepo.getTrendingGames(daysToMs(7));
+      });
+
+      afterAll(function () {
+        this.databaseClient.disconnect();
+        jasmine.clock().uninstall();
+      });
+
+      it("the resulting array has a length of 2", function () {
+        expect(this.result.length).toBe(2);
+      });
+
+      it("the first game is 'Risk of Strain'", function () {
+        expect(this.result[0].id).toBe(4);
+        expect(this.result[0].name).toBe("Risk of Strain");
+        expect(this.result[0].percentagePlayerIncrease).toBe(3350);
+      });
+
+      it("the second game is 'Risk of Brain'", function () {
+        expect(this.result[1].id).toBe(3);
+        expect(this.result[1].name).toBe("Risk of Brain");
+        expect(this.result[1].percentagePlayerIncrease).toBe(425.2);
+      });
+    });
+
+    describe("If a specific time period, return amount, and minimum players are passed in", function () {
+      beforeAll(async function () {
+        jasmine.clock().mockDate(new Date());
+
+        this.databaseClient = await initiateInMemoryDatabase(["games"]);
+
+        await this.databaseClient.insertMany("games", getTrendingGamesMockData());
+
+        const gamesRepo = new GamesRepository(this.databaseClient);
+
+        this.result = await gamesRepo.getTrendingGames(daysToMs(7), 1, 104);
+      });
+
+      afterAll(function () {
+        this.databaseClient.disconnect();
+        jasmine.clock().uninstall();
+      });
+
+      it("the resulting array has a length of 1", function () {
+        expect(this.result.length).toBe(1);
+      });
+
+      it("the first game is 'Risk of Brain'", function () {
+        expect(this.result[0].id).toBe(3);
+        expect(this.result[0].name).toBe("Risk of Brain");
+        expect(this.result[0].percentagePlayerIncrease).toBe(425.2);
+      });
+    });
+  });
 });
+
+const getTrendingGamesMockData = () => {
```

Comment:
Great! Very good job extracting that. Please open a new issue for a refatoring to put all mock data into separated modules next to the datamodel they mock. And add a todo to this function referencing the new issue.